README.md

```diff
@@ -1,16 +1,9 @@
 # Tesseract OCR
 
-For the latest online version of the README.md see:
-    
-https://github.com/tesseract-ocr/tesseract/blob/master/README.md
-  
-### Build 
-[![Build Status](https://travis-ci.org/tesseract-ocr/tesseract.svg?branch=master)](https://travis-ci.org/tesseract-ocr/tesseract)
-[![Build status](https://ci.appveyor.com/api/projects/status/miah0ikfsf0j3819/branch/master?svg=true)](https://ci.appveyor.com/project/zdenop/tesseract/)
-
-### Other
```

Comment:
Where? The idea behind this caption was to split badges.
In this PR they are again near each other.